### ISSUE TITLE
WCAG 2.2 1.4.6 Contrast (Enhanced) failure in WCAG 2.2 draft

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#3105]
I think there's a 1.4.6 Contrast (Enhanced) failure in the draft. 

The word NOTE in the note referring to Guideline 1.4 in 1.3.3 has contrast ratio 4.58
https://www.w3.org/TR/WCAG22/#sensory-characteristics

It's 12pt with text color: rgb(23,130,23) and background-color: rgb(233,251,233) and font-weight: 400 (normal)
https://contrast-ratio.com/#rgb%2823%2C130%2C23%29-on-rgb%28233%2C251%2C233

The other notes in the draft use the same style

**Comment by patrickhlauke** [comment]
I suspect this is an upstream issue on the generic W3C styles for all specs. changing the foreground colour from `hsl(120, 70%, 30%)` to `hsl(120, 70%, 22%)` will bump the contrast against `#E9FBE9` to `7.25:1`

**Comment by plehegar** [comment]
This got fixed in the W3C TR stylesheet. I believe this can be closed.

**Comment by patrickhlauke** [comment]
any idea when it will be pushed to prod?

**Comment by dd8** [comment]
> @patrickhlauke any idea when it will be pushed to prod?

I'm seeing the fixed version - but base.css has Cache-Control: max-age=604800 set in the HTTP headers so old version is probably cached for up to 604800 secs (7 days)

**Comment by dd8** [comment]
> @plehegar This got fixed in the W3C TR stylesheet see [w3c/tr-design#339](https://github.com/w3c/tr-design/pull/339) . I believe this can be closed.

Yes, I can see the fix and it has contrast 7.28, so I'm good with this, but @patrickhlauke can't see the fix. There could be an old version still on some Cloudflare CDN edge servers (the TR stylesheet has a 7 day max cache age)

**Comment by patrickhlauke** [comment]
purged my cache and i can see the fix. yes, we should be good to close this...which i'll do now :)
